projects/ngrx.io/content/marketing/docs.md

```diff
@@ -2,14 +2,57 @@
 
 NgRx is a framework for building reactive applications in Angular. NgRx provides state management, isolation of side effects, entity collection management, router bindings, code generation, and developer tools that enhance developers experience when building many different types of applications.
 
-## Core Principles
-
-- State is a single, immutable data structure.
-- Components delegate responsibilities to side effects, which are handled in isolation.
-- Type-safety is promoted throughout the architecture with reliance on TypeScript's compiler for program correctness.
-- Actions and state are serializable to ensure state is predictably stored, rehydrated, and replayed.
-- Promotes the use of functional programming when building reactive applications.
-- Provide straightforward testing strategies for validation of functionality.
+## Why NgRx?
+
+NgRx is built for creating maintainable explicit applications, by storing single state and the use of actions in order to express state changes.
+
+
+### Serializability
+
+By normalizing state changes and pass them through observables, NgRx provides serializability and ensures state is predictably stored. This enables to save the state to an external storage, for example, `localStorage`.
+
+In addition, it also allows to inspect, download, upload, and dispatch actions, all from the [Store Devtools](guide/store-devtools).
+
+
+### Type Safety
+Type safety is promoted throughout the architecture with reliance on the TypeScript compiler for program correctness.
+
+
+### Encapsulation
+
+Using NgRx [Effects](guide/effects) and [Store](guide/store), any interaction with external resources side effects, like network requests, web socket and any business logic can be isolated from the UI. This isolation allows for more pure and simple components, and keep the single responsibility principle.
+
+### Testable
+
+Since NgRx is mostly based on pure functions like `selectors` and `reducers`, plus the ability to isolate side effects from the UI, testing becomes very straightforward.
```

Comment:
```suggestion
Because [Store](guide/store) uses pure functions for changing state and selecting data from state, and the ability to isolate side effects from the UI, testing becomes very straightforward.
```